spec/system/transfer_system_spec.rb

```diff
@@ -107,6 +107,14 @@ def create_transfer(amount, from_name, to_name)
     expect(page).to have_no_text 'Inactive R Us'
   end
 
+  # 4438 - Bug Fix
```

Comment:
Ditto.

Reply:
There in no test case for errors in this file, so I created a new test case.